File: Copilot/sistemaDeTipos/tipos.js
function promedioMayoresAUmbral(precios, umbral) {
  let suma = 0;
  let cantidad = 0;

  for (const precio of precios) {
    if (precio > umbral) {
      suma += precio;
      cantidad++;
    }
  }

  return cantidad === 0 ? 0 : suma / cantidad;
}

File: Copilot/sistemaDeTipos/tipos.rs
fn promedio_mayores_a_umbral(precios: &[f64], umbral: f64) -> f64 {
    let mut suma = 0.0;
    let mut cantidad = 0usize;

    for &precio in precios {
        if precio > umbral {
            suma += precio;
            cantidad += 1;
        }
    }

    if cantidad == 0 {
        0.0
    } else {
        suma / cantidad as f64
    }
}